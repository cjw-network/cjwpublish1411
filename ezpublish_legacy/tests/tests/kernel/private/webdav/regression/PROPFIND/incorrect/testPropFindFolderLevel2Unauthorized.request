<?php

// Test if PROPFIND returns '401 Unauthorized' for the folder $GLOBALS['ezc_webdav_testfolder']
// with wrong username and password

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/Content/' . $GLOBALS['ezc_webdav_testfolder'] . '/';
$_SERVER['REQUEST_METHOD'] = 'PROPFIND';

$_SERVER['HTTP_DEPTH'] = '1';
$_SERVER['CONTENT_TYPE'] = 'application/xml';
$_SERVER['HTTP_CONTENT_LENGTH'] = '288';

// Set a wrong username and password which will be used in the main test
$GLOBALS['ezc_webdav_username'] = 'wrong_username';
$GLOBALS['ezc_webdav_password'] = 'wrong_password';

// The PROPFIND request needs an XML body
$GLOBALS['ezc_post_body'] = <<<END
<?xml version="1.0" encoding="utf-8"?>
<propfind xmlns="DAV:"><prop>
<creationdate xmlns="DAV:"/>
<getcontentlength xmlns="DAV:"/>
<getcontenttype xmlns="DAV:"/>
<getlastmodified xmlns="DAV:"/>
<displayname xmlns="DAV:"/>
<executable xmlns="http://apache.org/dav/props/"/>
<resourcetype xmlns="DAV:"/>
<supportedlock xmlns="DAV:"/>
<checked-in xmlns="DAV:"/>
<checked-out xmlns="DAV:"/>
</prop></propfind>
END;

?>
