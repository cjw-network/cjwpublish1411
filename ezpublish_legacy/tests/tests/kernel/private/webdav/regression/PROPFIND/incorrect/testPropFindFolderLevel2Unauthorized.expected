WWW-Authenticate: Basic realm="eZ Components WebDAV"
Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Type: text/plain; charset="utf-8"
Content-Length: 22
HTTP/1.1 401 Unauthorized

Authentication failed.
